src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,58 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { getMovie } from '../../api';
+// import { MovieData } from '../../types/MovieData';
+// import { ResponseError } from '../../types/ReponseError';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 import './FindMovie.scss';
 
-export const FindMovie: React.FC = () => {
+// const defaultMovie = {
+//   title: '',
+//   description: '',
+//   imgUrl: '',
+//   imdbUrl: '',
+//   imdbId: '',
+// };
+
+interface Props {
+  addMovie: (movie: Movie) => void;
```

Comment:
onMovieAdd

https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364